Penteract/Assets/Scripts/RangedAI.cpp

```diff
@@ -74,10 +76,6 @@ void RangedAI::Start() {
 	if (meshObj) {
 		ComponentBoundingBox* bb = meshObj->GetComponent<ComponentBoundingBox>();
 		bbCenter = (bb->GetLocalMinPointAABB() + bb->GetLocalMaxPointAABB()) / 2;
-		meshRenderer = meshObj->GetComponent<ComponentMeshRenderer>();
-		if (meshRenderer) {
-			noDmgMaterialID = meshRenderer->materialId;
-		}

```

Comment:
Are you sure this should be removed? The dissolve and hit animations aren't playing.

Reply:
That one is being set on the SetRandomMaterial();  once we know the random material 
noDmgMaterialID = materials[position];

Reply:
Yes, but what about meshRenderer?

Reply:
You are right . Fixed